census/src/main/java/io/grpc/census/CensusStatsAccessor.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2019 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.census;
+
+import com.google.common.base.Stopwatch;
+import com.google.common.base.Supplier;
+import io.grpc.ClientInterceptor;
+import io.grpc.ServerStreamTracer;
+
+/**
+ * Accessor for getting {@link ClientInterceptor} or {@link ServerStreamTracer.Factory} with
+ * default Census stats implementation.
+ */
+public final class CensusStatsAccessor {
```

Comment:
Let's rename this to `InternalCensusStatsAccessor` and mark it `@Internal`. It looks like we'll need to do a resonable amount of work before we can use zero-arg (or almost zero) methods to construct the objects here. Once we get to zero (or almost zero) we can make something here public or use the package-private+public-inner-class trick.